test/__snapshots__/loader.test.js.snap

```diff
@@ -96,7 +96,7 @@ exports[`null-loader should works 1`] = `
 /*! no static exports found */
 /***/ (function(module, exports) {
 
-// empty (null-loader)
+/* empty (null-loader) */
```

Comment:
It is still `js` assets, so `//` also works. I think you need testing this in other way:
1. Create css file
2. Create custom `url()` loader (very simple implementation)
3. Create custom css loader (very simple implementation)

Input:
```
a {
  background: url(path/to/something) no-repeat;
}
```

Output:
```
a {
  background: /* comment */ no-repeat;
}
```